dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -35,7 +46,7 @@ new RenderingCDKStack(cdkApp, 'ArticleRendering-CODE', {
 	stage: 'CODE',
 	domainName: 'article-rendering.code.dev-guardianapis.com',
 	scaling: { minimumInstances: 1, maximumInstances: 3 },
-	instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
+	instanceType: InstanceType.of(InstanceClass.C7G, InstanceSize.MEDIUM),
```

Comment:
Why are we changing this?

Reply:
Good spot thank you! A mistake left over from testing in CODE.